tests/test_control_traffic.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2016-2021 XMOS LIMITED.
+# This Software is subject to the terms of the XMOS Public Licence: Version 1.
+from usb_packet import (
+    TokenPacket,
+    TxDataPacket,
+    RxDataPacket,
+    TxHandshakePacket,
+    RxHandshakePacket,
+    USB_PID,
+)
+from usb_session import UsbSession
+from usb_transaction import UsbTransaction
+import pytest
+from conftest import PARAMS, test_RunUsbSession
+from copy import deepcopy
+
+# Only test on EP 0 - Update params
+PARAMS = deepcopy(PARAMS)
+PARAMS["default"].update({"ep": [0]})
+PARAMS["smoke"].update({"ep": [0]})
```

Comment:
Does this need to update "extended" as well? As per your other PR:
```
for v in PARAMS.values():
    v.update({"ep": [0]})
```

Reply:
ah yes, thanks!